app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -0,0 +1,30 @@
+class Users::OmniauthCallbacksController < Devise::OmniauthCallbacksController
```

Comment:
This controller will need to be moved into the api/v1 module once our changes are brought together.